src/cmd/validate-templates.ts

```diff
@@ -186,8 +186,8 @@ export const validateTemplates = async (): Promise<void> => {
     })
   output.PASS?.map((_val: string) => debug.info(`${chalk.greenBright('PASS: ')} ${chalk.italic('%s')}`, _val))
   output.WARN?.map((_val: string) => debug.warn(`${chalk.yellowBright('WARN: ')} %s`, _val))
-  if (kubevalOutput.exitCode !== 0 || output.ERR) {
-    output.ERR?.map((_val: string) => debug.error(`${chalk.redBright('ERR: ')} %s`, _val))
+  if (kubevalOutput.exitCode !== 0) {
+    if (output.ERR) output.ERR?.map((_val: string) => debug.error(`${chalk.redBright('ERR: ')} %s`, _val))
```

Comment:
We don't need to check for this, as there is already `output.ERR?` where the `?` makes it nullable, thus not running map.
Reason `|| output.ERR` was in the if statement is because in the "old CLI" it exits with exit code 1, if the output contains an error: https://github.com/redkubes/otomi-core/blob/master/bin/validate-templates.sh#L109

So this would be a regression compared to the old functionality

Reply:
seems we can remove ` if (output.ERR) ` statement